Sources/KituraNet/FastCGI/FastCGIServer.swift

```diff
@@ -53,17 +57,21 @@ public class FastCGIServer: Server {
     /// Listens for connections on a socket
     ///
     /// - Parameter on: port number for new connections
-    public func listen(on port: Int) throws {
+    /// - Parameter address: The address of the network interface to listen on. Defaults to nil, which means this
+    ///             server will listen on all interfaces.
+    public func listen(on port: Int, address: String?) throws {
```

Comment:
Although FastCGI is not implemented yet, the function signature should still be consistent with Kitura-net (as per @pushkarnk 's comments on HTTPServer) - need to add the default `nil` value.